src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/codeAction/proposal/quickfix/InsertAnnotationQuickFix.java

```diff
@@ -112,4 +131,16 @@ protected PsiModifierListOwner getBinding(PsiElement node) {
     protected String getLabel(String annotationName, String... attributes) {
         return Messages.getMessage("InsertItem", "@" + annotation); // uses Java syntax
     }
+
+    private CodeAction createCodeAction(JavaCodeActionContext context, Diagnostic diagnostic, String label) {
```

Comment:
@mrglavas - can we raise a separate PR for moving this method to a common file?

Reply:
@aparnamichael Yes. Feel free to open a separate issue and PR for this. Thanks.

Reply:
Created PR for the same - https://github.com/OpenLiberty/liberty-tools-intellij/pull/531